versioned_docs/version-latest/community/index.md

```diff
@@ -11,7 +11,7 @@ keywords:
 
 :::caution
 
-Please note, all community provided resources are **non-official**. For clarification please refer to [official materials](../intro).
+Please note, all community provided resources are **non-official**. For clarification please refer to [official materials](/docs/intro).
```

Comment:
This link ignores the selected documentation version,  for the latest version, in this case, I believe it should be along the lines of `(/versioned-docs/version-latest/intro)`. Please make sure to test it.

Reply:
There may be some issues since I'm not really familiar with this repo very well. Btw, do we need this changes now? Seems we have figure out the issue and can be fixed in other ways.